doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1475,6 +1475,90 @@ The Opcodes Aggregator client is an optional Bluetooth mesh model that can be en
 	Abort the Opcodes Aggregator Sequence message. This command clears the Opcodes Aggregator Client context.
 
 
+Remote Provisioning Client
+--------------------------
+
+The Remote Provisioning client is an optional Bluetooth mesh model that can be enabled through the :kconfig:option:`CONFIG_BT_MESH_RPR_CLI` configuration option. The Remote Provisioning Client model is used to support the functionality of dispatching a sequence of access layer messages to nodes supporting the Remote Provisioning Server model.
+
+``mesh models rpr scan <timeout in seconds> [<UUID>]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Must be at least 1 second.
+	* ``UUID``: Device UUID to scan for, or NULL to report all devices.
+
+``mesh models rpr scan-ext <timeout in seconds> <UUID> [<AD-type> ... ]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start extended scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Valid values from @ref BT_MESH_RPR_EXT_SCAN_TIME_MIN to @ref BT_MESH_RPR_EXT_SCAN_TIME_MAX.
```

Comment:
Is there a reason for having @ref before the two values?
Are these config options, could it be written :kconfig:option: instead?
If not, enclose with ``.

Reply:
These are not kconfig options.